### PR TITLE
Rv asfmt 346

### PR DESCRIPTION
**Pull request by robertvazquezlegalshield**
This is to confirm that no results are displayed when entering bad search information.  The browser pause is to allow the the browser plenty of time to load the graphic used to confirm no results are available.

### CODE REVIEW

File: demo/adonis/partner/no_results_with_bad_search_input.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var i = require('../../tools.js')
+var secure = require('../../secret.js')
+
+who('As a Partner I can', function() {
+where('https://partner.sandbox.legalshield.com')
+what('see no results with bad input ASMFT-346', function () {
+  click('mat-form-field.mat-form-field:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1)')
+    enterText('#mat-input-0', secure.partner_login)
```
Comment (TrentonWalkup) [inline demo/adonis/partner/no_results_with_bad_search_input.js]: we don't need to indent here. can keep all of these 'enterText' actions aligned with the 'click'

Comment (robertvazquezlegalshield) [reply demo/adonis/partner/no_results_with_bad_search_input.js]: I've corrected this now.

**Comment by Gregorma** [comment]
LGTM

**Comment by TrentonWalkup** [comment]
hey, you have conflicts

**Comment by TrentonWalkup** [comment]
you have to resolve them.

**Comment by robertvazquezlegalshield** [comment]
Conflicts have been resolved

**Comment by Gregorma** [comment]
LGTM

**Pull request merged**